### PR TITLE
[IOTDB-215]Update TsFile sketch tool and TsFile docs for v0.9.0

### PR DESCRIPTION
**Pull request by LeiRui**

### CODE REVIEW

File: docs/Documentation/UserGuide/8-System Design (Developer)/1-Hierarchy.md [hunk 1]
```diff
@@ -123,30 +120,30 @@ PageHeader Structure
         	<tr><td>Data size before compressing</td><td>int</td>
         	<tr><td>Data size after compressing(if use SNAPPY)</td><td>int</td>
         	<tr><td>Number of values</td><td>int</td>
-        	<tr><td>Minimum time stamp</td><td>long</td>
         	<tr><td>Maximum time stamp</td><td>long</td>
-        	<tr><td>Minimum value of the page</td><td>Type of the page</td>
+        	<tr><td>Minimum time stamp</td><td>long</td>
         	<tr><td>Maximum value of the page</td><td>Type of the page</td>
+          <tr><td>Minimum value of the page</td><td>Type of the page</td>
```
Comment (jt2594838) [inline docs/Documentation/UserGuide/8-System Design (Developer)/1-Hierarchy.md]: Beware of the formats.

File: docs/Documentation/UserGuide/8-System Design (Developer)/1-Hierarchy.md [hunk 2]
```diff
@@ -296,92 +290,154 @@ If size of props is greater than 0, there is an array of <String, String> pair a
...
 ```
 
-## A TsFile Visualization Example (v0.8.0)
-![A TsFile Visualization Example v0.8.0](https://user-images.githubusercontent.com/33376433/65209576-2bd36000-dacb-11e9-9e43-49e0dd01274e.png)
+#### 1.3.3 TsFileSequenceRead
+You can also use `example/tsfile/org/apache/iotdb/tsfile/TsFileSequenceRead` to sequentially print a TsFile's content.
+
+### 1.4 A TsFile Visualization Example (v0.8.0)
 
+![A TsFile Visualization Example v0.8.0](https://user-images.githubusercontent.com/33376433/65209576-2bd36000-dacb-11e9-9e43-49e0dd01274e.png)
```
Comment (jt2594838) [inline docs/Documentation/UserGuide/8-System Design (Developer)/1-Hierarchy.md]: Should this v0.8.0 be updated?

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBQueryDemoIT.java [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+    Class.forName(Config.JDBC_DRIVER_NAME);
+    try (Connection connection = DriverManager
+        .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
+        Statement statement = connection.createStatement()) {
+
+      for (String sql : sqls) {
+        statement.execute(sql);
+      }
+    } catch (Exception e) {
+      e.printStackTrace();
```
Comment (jt2594838) [inline server/src/test/java/org/apache/iotdb/db/integration/IoTDBQueryDemoIT.java]: You should fail it if there does be an exception.

File: docs/Documentation/UserGuide/8-System Design (Developer)/1-Hierarchy.md (hunk 1 above)
Comment (LeiRui) [reply docs/Documentation/UserGuide/8-System Design (Developer)/1-Hierarchy.md]: fixed

File: docs/Documentation/UserGuide/8-System Design (Developer)/1-Hierarchy.md (hunk 2 above)
Comment (LeiRui) [reply docs/Documentation/UserGuide/8-System Design (Developer)/1-Hierarchy.md]: yes

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBQueryDemoIT.java (hunk 3 above)
Comment (LeiRui) [reply server/src/test/java/org/apache/iotdb/db/integration/IoTDBQueryDemoIT.java]: fixed

**Pull request merged**